common-canvas/expression/function-list-v3-schema.json

```diff
@@ -49,7 +49,7 @@
         "label": {
           "description": "External name of category",
           "type": "object",
-          "$ref": "https://github.com/elyra-ai/pipeline-schemas/blob/main/common-pipeline/operators/uihints-v3-schema.json#/definitions/resource_definition"
+          "$ref": "https://api.dataplatform.ibm.com/schemas/common-pipeline/operators/uihints-v3-schema.json#/definitions/resource_definition"
```

Comment:
Is there a reason we're using this URL?  It's not a valid URL.

Reply:
@tomlyn mentioned it is just a placeholder. This URL is used in several files. In the `generateTS.sh` script, we're using following code to replace this URL with relative path -

![Screenshot 2025-05-14 at 12 24 49 PM](https://github.com/user-attachments/assets/dbeedfae-a637-459f-9f9d-e6057947885f)

Reply:
Long term we probably should use valid URLs.